### DISCUSSION TITLE
graphscope coordinator pod lacks some python pkgs for Graph Learning

### DISCUSSION THREAD

**Discussion by Hspix** [alibaba/GraphScope#1021] [Q&A] [answered]
I use jupyterlab on jupyter container of graphScope coordinator pod, 
which tests for GCN:

**from graphscope.learning.examples import GCN**
and
**ModuleNotFoundError: No module named 'tensorflow'**

graphscope 0.8.0 is deployed by helm on k8s. 
Can add these pkgs for graph deep learning within deployment?

**Comment by sighingnow** [comment, marked as answer]
Thanks for reporting! Tensorflow is not required for the coordinator. GLE is responsible for sampling, the training process happens on the client side.

Closed as invalid.
